config.sample.yaml

```diff
@@ -76,3 +76,6 @@ developer_settings:
 
 # Used by the API and Resource processor application to change log level
 #  debug: true
+
+# Specify here tags that should be applied to all resources deployed by the TRE in JSON string format
+#  tags: '{"tag_key":"tag_value"}'
```

Comment:
Can you add this to the docs, https://microsoft.github.io/AzureTRE/v0.13.0/tre-admins/environment-variables/,

Can you also update the CHANGELOG.md file please.

Noticed this page needs updating to explain how `config.yaml` maps to ENV vars etc. Will create a separate issue for that.

Reply:
I've added this to the docs, can you verify it's correct. Thanks.